### PR TITLE
[Android] custom-easy filter @open sesame 7/7 15:16

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Update custom-filter API using custom-easy.
- register custom-filter with in/out info and implement invoke interface only.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestCustomFilter.java [hunk 1]
```diff
@@ -286,18 +277,58 @@ public TensorsData invoke(TensorsData in) {
...
+                }
+            });
 
-                    @Override
-                    public TensorsData invoke(TensorsData in) {
-                        return in;
-                    }
+            fail();
+        } catch (Exception e) {
+            /* expected */
```
Comment (helloahn) [inline api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestCustomFilter.java:296]: does it mean the expected exception?

Comment (jaeyun-jung) [reply api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestCustomFilter.java:296]: yes, right.

File: api/android/api/src/main/java/org/nnsuite/nnstreamer/CustomFilter.java [hunk 2]
```diff
@@ -65,20 +52,24 @@
...
      *
      * @return {@link CustomFilter} instance
      *
      * @throws IllegalArgumentException if given param is null
      * @throws IllegalStateException if failed to initialize custom-filter
      */
-    public static CustomFilter registerCustomFilter(@NonNull String name, @NonNull CustomFilterCallback callback) {
-        return new CustomFilter(name, callback);
+    public static CustomFilter registerCustomFilter(@NonNull String name, @NonNull TensorsInfo in,
+            @NonNull TensorsInfo out, @NonNull CustomFilterCallback callback) {
```
Comment (again4you) [inline api/android/api/src/main/java/org/nnsuite/nnstreamer/CustomFilter.java:71]: If Android API do not follow the backward compatibility, then it's OK.

Comment (myungjoo) [reply api/android/api/src/main/java/org/nnsuite/nnstreamer/CustomFilter.java:71]: It should be ok for now as we do not have corresponding applications, yet.

**Pull request merged**